bugbug/bug_features.py

```diff
@@ -594,6 +618,19 @@ def __call__(self, bug, **kwargs):
         return bug["status"]
 
 
+class HasOpenNeedinfoOnReporter(SingleBugFeature):
+    name = "Has Open Needinfo on Reporter"
+
+    def __call__(self, bug, **kwargs):
+        reporter_email = bug["creator_detail"]["email"]
+
+        return any(
+            flag["status"] == "?" and flag["requestee"] == reporter_email
+            for flag in bug["flags"]
+            if flag["name"] == "needinfo"
```

Comment:
I would move `flag["name"] == "needinfo"` to be part of the condition instead of having it in an if-statement.